lib/_tls_common.js

```diff
@@ -75,7 +75,7 @@ function SecureContext(secureProtocol, secureOptions, minVersion, maxVersion) {
   if (secureOptions) this.context.setOptions(secureOptions);
 }
 
-function validateKeyCert(name, value) {
+function validateKeyOrCertOption(name, value) {
   if (typeof value !== 'string' && !isArrayBufferView(value)) {
     throw new ERR_INVALID_ARG_TYPE(
```

Comment:
For another PR perhaps, especially since it will be semver-major, but it seems like this should be `ERR_INVALID_OPT_VALUE` instead.

Reply:
Nah, I'm wrong, it's the type that's invalid, not the value.